### PR TITLE
migrated opcodes tests for assembler from testcode to test...

### PR DESCRIPTION
**Pull request by SvOlli**
... for inclusion on automated testing

### CODE REVIEW

**Comment by greg-king5** [comment]
I think that you should take credit for your hard work.  Put a date and your name at the top of the assembly files.

File: test/assembler/Makefile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+TARGETS  = 6502 6502x 65sc02 65c02
+#TARGETS += 65816
+TARGETS += 4510
+TARGETS += huc6280
+#TARGETS += m740
+
+all: $(addprefix $(WORKDIR)/, $(addsuffix -opcodes.bin, $(TARGETS)))
+	@#
+
+.PHONY: $(addprefix $(WORKDIR)/, $(addsuffix -opcodes.bin, $(TARGETS)))
```
Comment (greg-king5) [inline test/assembler/Makefile]: `all` and `clean`, also, should be `.PHONY` goals.

**Comment by SvOlli** [comment]
I think credit for the assembler files is overrated here. ;-)

But adding a readme with some detailed explanation would be a good thing to do.

**Comment by SvOlli** [comment]
Added README and updated Makefile

**Comment by oliverschmidt** [comment]
As far as I understand this is now ready to be merged...

**Pull request merged**